### ISSUE TITLE
Bootstrap error: TypeError: node_fetch_1.default is not a function

### ISSUE THREAD

**Issue by chrissimpkins** [rust-analyzer/rust-analyzer#5267]
VS Code
Version: 1.45.1
Commit: 5763d909d5f12fe19f215cbfdd29a91c0fa9208a
Date: 2020-05-17T22:31:53.325Z
Electron: 7.1.14
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Linux x64 5.6.16-1-MANJARO

rust-analyzer worked in VS Code without issues in this GNU/Linux environment before my attempt to update the language server today.  There were no plugin configuration changes.  On the attempt to download and install Language server version 0.3.234-nightly:

```
INFO [7/8/2020, 1:20:50 PM]: Extension version: 0.3.234-nightly
INFO [7/8/2020, 1:20:50 PM]: Using configuration {
  diagnostics: { enable: true, warningsAsInfo: [], warningsAsHint: [] },
  lruCapacity: null,
  files: { watcher: 'client', exclude: [] },
  notifications: { cargoTomlNotFound: true },
  cargo: {
    noDefaultFeatures: false,
    allFeatures: false,
    features: [],
    loadOutDirsFromCheck: false,
    target: null
  },
  rustfmt: { extraArgs: [], overrideCommand: null },
  checkOnSave: {
    enable: true,
    extraArgs: [],
    command: 'check',
    overrideCommand: null,
    allTargets: true,
    allFeatures: null,
    features: null
  },
  cargoRunner: null,
  runnableEnv: null,
  inlayHints: {
    enable: true,
    typeHints: true,
    chainingHints: true,
    parameterHints: true,
    maxLength: 20
  },
  completion: {
    addCallParenthesis: true,
    addCallArgumentSnippets: true,
    postfix: { enable: true }
  },
  callInfo: { full: true },
  updates: { channel: 'nightly', askBeforeDownload: true },
  serverPath: null,
  trace: { server: 'off', extension: false },
  procMacro: { enable: false },
  debug: {
    engine: 'auto',
    sourceFileMap: {
      '/rustc/<id>': '${env:USERPROFILE}/.rustup/toolchains/<toolchain-id>/lib/rustlib/src/rust'
    },
    openDebugPane: false,
    engineSettings: {}
  },
  lens: { enable: true, run: true, debug: true, implementations: true },
  hoverActions: {
    enable: true,
    implementations: true,
    run: true,
    debug: true,
    gotoTypeDef: true
  },
  linkedProjects: null,
  withSysroot: true
}
INFO [7/8/2020, 1:20:50 PM]: PersistentState: {
  lastCheck: 1594228772959,
  releaseId: 28336980,
  serverVersion: '0.3.233-nightly'
}
ERROR [7/8/2020, 1:21:05 PM]: Bootstrap error [TypeError: node_fetch_1.default is not a function
	at Object.fetchRelease (/home/chris/.vscode-oss/extensions/matklad.rust-analyzer-0.3.234-nightly/out/src/main.js:15902:48)
	at getServer (/home/chris/.vscode-oss/extensions/matklad.rust-analyzer-0.3.234-nightly/out/src/main.js:16235:31)
	at async bootstrapServer (/home/chris/.vscode-oss/extensions/matklad.rust-analyzer-0.3.234-nightly/out/src/main.js:16145:18)
	at async bootstrap (/home/chris/.vscode-oss/extensions/matklad.rust-analyzer-0.3.234-nightly/out/src/main.js:16095:18)
	at async tryActivate (/home/chris/.vscode-oss/extensions/matklad.rust-analyzer-0.3.234-nightly/out/src/main.js:16022:24)
	at async activate (/home/chris/.vscode-oss/extensions/matklad.rust-analyzer-0.3.234-nightly/out/src/main.js:15996:5)]
```

I receive the error dialog:

```
Cannot activate rust-analyzer: bootstrap error.
```

Please let me know if you need more detailed information.

**Comment by bjorn3** [comment]
Duplicate of #5257

Fixed in #5262

**Comment by chrissimpkins** [comment]
Apologies! Thank you very much
